### PR TITLE
Collections design review 3

### PR DESCRIPTION
**Pull request by mikeger**
- Allow saving of images and copying
- Auto-open downloaded files/assets
- Correct dismissal of full screen image view
- Put videos after images
- Correct collections view insets

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Components/Controllers/FullscreenImageViewController.m [hunk 1]
```diff
@@ -299,7 +309,7 @@ - (void)setupGestureRecognizers
     [self.scrollView addGestureRecognizer:panRecognizer];
     
     [self.doubleTapGestureRecognizer requireGestureRecognizerToFail:panRecognizer];
-    [self.longPressGestureRecognizer requireGestureRecognizerToFail:panRecognizer];
+//    [self.longPressGestureRecognizer requireGestureRecognizerToFail:panRecognizer];
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Components/Controllers/FullscreenImageViewController.m]: Delete

Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Components/Controllers/FullscreenImageViewController.m]: Can be removed?

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Components/Controllers/FullscreenImageViewController.m]: Actually should be put back. Thanks!

**Comment by mikeger** [comment]
@daehn @typfel updated

**Comment by mikeger** [comment]
Let me run the tests

**Pull request merged**